### ISSUE TITLE
Formatting removes alignment markers if there are leading/trailing spaces in the 2nd line

### ISSUE THREAD

**Issue by roman-r-m** [darkriszty/MarkdownTablePrettify-VSCodeExt#60]
Formatting this table
```
| A   | B   | C   | D   |
| --- | ---: | :---: | :--- |
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

produces
```
| A        | B        | C       | D     |
|----------|----------|---------|-------|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

when in fact it should be something like this
```
| A        |        B |    C    | D     |
|----------|---------:|:-------:|:------|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |

```

**Comment by roman-r-m** [comment]
Changing the input to (i.e. removing spaces in the 2nd line):
```
| A   | B   | C   | D   |
|-----|----:|:---:|:---|
| dhdhgfgh | dsgsdgdg | dsgsdfg | zxczc |
```

fixes the issue.

From a quick glance at the code it might be enough to strip leading & trailing whitespaces in this method
https://github.com/darkriszty/MarkdownTablePrettify-VSCodeExt/blob/6590028b6741bbb2ae8e88b2c1972438f40af1fa/src/modelFactory/alignmentFactory.ts#L9

**Comment by darkriszty** [comment]
Thanks for reporting and also looking into the code 👍 I will make the adjustments and include them in the next release🙂

**Comment by darkriszty** [comment]
The change is available in 3.5.0, let me know if you run into issues.

**Comment by roman-r-m** [comment]
Thank you for fixing this quickly!
